workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
Was this interface defined by POP-C++? Because in Java that would be wrong. You can't pass floats as out parameters to a function.

If it is, then i propose the following. Rename POPFloat to POPMutableFloat to make clear what it is. That way it will be clear that its very different to normal floats (and Floats) in Java.